api/mock_server/app.py

```diff
@@ -107,32 +107,32 @@ def get_submission(token_info, course_id, submission_id, poll=False):
                  "code": "client_error"}, 404)
     return (submission_info, 200)
 
-def get_submission_list(token_info, course_id, task_id):
+def get_submission_list(token_info, course_id, task):
     print(f"token_info: {token_info}")
     print(f"course_id: {course_id}")
-    print(f"task_id: {task_id}")
-    return ({"submissions:": [
+    print(f"task_id: {task}")
```

Comment:
There should probably exist a case on the test server where the list of submissions is empty.

```suggestion
    print(f"task_id: {task}")
    if task == 404:
        return ({"submissions": []}, 200)
```